src/Payum/Core/Bridge/Propel/Storage/PropelStorage.php

```diff
@@ -0,0 +1,88 @@
+<?php
+namespace Payum\Core\Bridge\Propel\Storage;
+
+use Payum\Core\Storage\AbstractStorage;
+use Payum\Core\Model\Identificator;
+
+class PropelStorage extends AbstractStorage
+{
+    /**
+     * @var string
+     */
+    protected $modelQuery;
+    /**
+     * @var string
+     */
+    protected $modelPeer;
+
+    /**
+     * @param string $modelClass
+     */
+    public function __construct($modelClass)
+    {
+        parent::__construct($modelClass);
+
+        $this->modelQuery = $modelClass . 'Query';
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function findModelById($id)
+    {
+        $modelQuery = $this->modelQuery;
+        return $modelQuery::create()
+            ->findPk($id);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function doUpdateModel($model)
+    {
+        $model->save();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function doDeleteModel($model)
+    {
+        $model->delete();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function doGetIdentificator($model)
+    {
+        $id = $this->getModelId($model);
+
+        if (count($id) > 1) {
+            throw new \LogicException('Storage not support composite primary ids');
+        }
+
+        return new Identificator(array_shift($id), $model);
```

Comment:
Identity now, many of storage methods were  renamed in last version. so the code must be rebased.